src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StartedNode.kt

```diff
@@ -66,11 +66,13 @@ class StartedNode(
             jiraHome = this.jiraHome,
             analyticLogs = analyticLogs,
             resultsTransport = this.resultsTransport,
-            unpackedProduct = this.unpackedProduct,
+            jiraPath = this.jiraPath,
             monitoringProcesses = this.monitoringProcesses,
             ssh = this.ssh
         )
     }
 
     override fun toString() = name
-}
\ No newline at end of file
+
+    fun toStoppableNode(): StoppableNode = StoppableNode.Builder(jiraPath, ssh).build()
```

Comment:
What's the reason for not making this "stoppability" a feature of `StartedNode`? I'm not sure if I understand the code modeling strategy here.